kubernetes/cray-dhcp-kea/files/startup-dhcp.sh

```diff
@@ -7,6 +7,17 @@ BACKUP_CONFIG_PATH=/srv/kea/backup/
 BACKUP_CONFIG_FILE=keaBackup.conf.gz
 KEA_CONFIG_PATH=/usr/local/kea/
 
+#  check username and passward are empty
+#  this can happen if the pod restarts
+if [ -z "$DHCP_DBUSER" ]
+then
+      DHCP_DBUSER=$(cat /secrets/postgres/dhcpdsuser/username)
+fi
+if [ -z "$DHCP_DBPASS" ]
+then
+      DHCP_DBPASS=$(cat /secrets/postgres/dhcpdsuser/password)
+fi
```

Comment:
What are these env variables being used for? Looking at your changes to this file I don't see where they are used, unless I am missing something.

Reply:
They are used inside dhcp_helper